toolkit/syno-evansport-6.2.2/Makefile

```diff
@@ -0,0 +1,16 @@
+TOOLKIT_NAME = syno-$(TOOLKIT_ARCH)
+
+TOOLKIT_ARCH = evansport
+TOOLKIT_VERS = 6.2
+
+TOOLKIT_DIST = ds.evansport-$(TOOLKIT_VERS).dev
+TOOLKIT_EXT = txz
+TOOLKIT_DIST_NAME = $(TOOLKIT_DIST).$(TOOLKIT_EXT)
+TOOLKIT_DIST_SITE = https://sourceforge.net/projects/dsgpl/files/toolkit/DSM$(TOOLKIT_VERS)
+
+TOOLKIT_PREFIX = local
+TOOLKIT_STRIP = 6
+TOOLKIT_BASE_DIR = i686-pc-linux-gnu
+TOOLKIT_SYSROOT = $(TOOLKIT_BASE_DIR)/sys-root/usr
```

Comment:
To avoid copy-paste, I would define most of these variables as "default value" in toolkit mk (except `TOOLKIT_BASE_DIR`)
For instance with `TOOLKIT_DIST = ds.$(TOOLKIT_ARCH)-$(TOOLKIT_VERS).dev`

So if variable is not already set (override specifically by arch toolkit), toolkit makefile will set them with defaults from others.

Reply:
Good point.  This work was based on `TC_` in its original form prior to #4142.
I'll make the necessary changes to re-use some of the work done for `TC_`

Reply:
Done up to the maximum I could get with commits https://github.com/SynoCommunity/spksrc/pull/4143/commits/a61f004a4df645b536e10d6641870588f591f6b3 and https://github.com/SynoCommunity/spksrc/pull/4143/commits/5510d19c87769489aa00980339351b64d95625e3